index.html

```diff
@@ -1,78 +1,72 @@
-            <p>Helping Spanish Speakers Understand Web Development Terminology</p>
-            <p>Ayudando a los Hispanohablantes a Aprender Terminolog&iacute;a de Desarollo Web</p>
-        </div>
-    </header>
-    <div class="nav-placeholder" style="height: 42px;">
-        <nav class="">
-            <div class="nav-inner">
-                <div class="nav-inner-most">
-                    <ul class="clearfix">
-                        <li>
-                            <form action="form1">
-                                <input type="text" name="FirstName" value="Place holder">
-                                <br>
-                            </form>
-                        </li>
-                    </ul>
-                </div>
-            </div>
-        </nav>
+  <header class="site-header">
+    <div class="site-header-inside inside">
+      <div class="site-title">
+        <a href="/LMHT" id="logo">
+          <img src="img/lmht-logo.svg" alt="LMHT Logo">
+        </a>
+      </div>
     </div>
-    <h4 class='searchlabel'>Select an element to view definition, memorization aids, audio pronunciation, and code sample:<br>Selecione un elemento para ver la definici&oacute;n, ayuda para recordar, la pronunciaci&oacute;n con audio, y ejemplos de como usar la etiqueta:</h4>
-    <div class='droplist'>
-        <label for='tags'></label>
-        <select name='tags' id='tags'>
-            <option selected value="empty.html">Select a tag/Selecione una etiqueta</option>
-            <option value='a.html'>&lt;a></option>
-            <option value='base.html'>&lt;base></option>
-            <option value='body.html'>&lt;body></option>
-            <option value='br.html'>&lt;br></option>
-            <option value='footer.html'>&lt;footer></option>
-            <option value='head.html'>&lt;head&gt;</option>
-            <option value='header.html'>&lt;header></option>
-            <option value='html.html'>&lt;html></option>
-            <option value='link.html'>&lt;link></option>
-            <option value='meta.html'>&lt;meta&gt;</option>
-            <option value='style.html'>&lt;style></option>
-            <option value='title.html'>&lt;title></option>
-        </select>
+    <div class="header-meta">
+      <p>Helping Spanish Speakers Understand Web Development Terminology</p>
+      <p>Ayudando a los Hispanohablantes a Aprender Terminolog&iacute;a de Desarollo Web</p>
     </div>
-    <main>
-        <div class="main-inside inside" id="main">
+  </header>
+  <div>
+    <h4 class='searchlabel'>Select an element to view definition, memorization aids, audio pronunciation, and code sample:<br>Selecione un elemento para ver la definici&oacute;n, ayuda para recordar, la pronunciaci&oacute;n con audio, y ejemplos de como usar la etiqueta:</h4>
+  </div>
+  <div class="sticky-combo">
+    <nav class="">
+      <div class="nav-inner">
+        <div class="nav-inner-most">
+          <ul class="clearfix">
```

Comment:
This is still super nested and I don't immediately see the purpose. Also, see my other comment.

At the very least, you don't need `class=""` on that `nav`.